doc/api/util.md

```diff
@@ -3305,6 +3305,29 @@ util.isUndefined(null);
 // Returns: false
 ```
 
+### `util.loadEnvFile(path)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.1 - Active Development
+
+* `path` {string | undefined}
+
+Loads the `.env` file into `process.env`. By default the path is resolved to
+`path.resolve(process.cwd(), '.env')`. If file is not found, no error is thrown.
```

Comment:
If the file is not found, it should throw an error. It’s trivial to try/catch this, why should it fail silently?

Reply:
I think we failed to follow the general use case. `Dotenv` doesn't throw an error instead warns the user: https://github.com/motdotla/dotenv/blob/master/lib/main.js#L228

I recommend not throwing an error.